backend/src/logUtils.ts

```diff
@@ -34,31 +34,30 @@ const cropAndLogMessage = (txt: string) => {
 };
 
 export function logMessage({
-  eloId,
+  idToken,
   socketId,
   game,
   sent,
   messageTitle,
   messageParams,
 }: {
-  eloId: string | null;
+  idToken: string | null;
   socketId: string;
   game: GameState | null;
   sent: boolean;
   messageTitle: string;
   messageParams: any;
 }) {
-  let shortEloId = eloId ? eloId.substring(0, 3) : "";
-  while (shortEloId.length < 3) shortEloId = shortEloId + "_";
+  let shortIdToken = idToken ? idToken.substring(6, 9) : ""; // starts at index 6 to not include "Auth0|"
```

Comment:
I think instead of hardcoding 6 here and explaining in a comment why, it would be cleaner to have a variable "auth0Prefix: string = "Auth0|", and then use auth0Prefix.length instead of 6.

Reply:
ok sure

Reply:
done